Src/Website/docusaurus.config.js

```diff
@@ -110,9 +109,9 @@ const config = {
                 ],
             },
             prism: {
-                additionalLanguages: ["csharp"],
-                theme: lightCodeTheme,
-                darkTheme: darkCodeTheme,
+                additionalLanguages: ["csharp", "bash", "powershell", "json", "editorconfig", "yaml", "ignore"],
+                theme: prismReactRenderer.themes.github,
```

Comment:
Consider if this is the correct theme to use? Its a bit bright and low contrast for my eyes, but this is at least the same as you had before.

Reply:
I'm not opposed to it being changed, I think I just based the current one off of how the prettier site looked

Reply:
That reasoning works for me! I'd consider using the vsLight theme for increased contrast, but this is all preference so lets merge this and if you want to switch just do it separately 👍

Reply:
vsLight looked pretty similar to me. I apparently disabled the toggle that lets the user enable dark mode. So instead of changing the theme I brought that back and got it looking good. Going back to the light mode after seeing it with dark mode is tough on my eyes

Reply:
Agreed! I just assumed there was a reason for the no dark mode since it was disabled 🌞

The dark themes look better, so I support enabling it! 👍